src/daemon/daemon.cpp

```diff
@@ -1680,6 +1680,21 @@ try // clang-format on
             delayed_shutdown_instances.erase(name);
         }
 
+        try
+        {
+            auto& sshfs_mounts = mount_threads.at(name);
+            for (const auto& sshfs_mount : sshfs_mounts)
+            {
+                mpl::log(mpl::Level::debug, "delete",
+                         fmt::format("Stopping mount '{}' in instance \"{}\"", sshfs_mount.first, name));
+                sshfs_mount.second->stop();
+            }
+        }
+        catch (const std::exception& e)
+        {
+            mpl::log(mpl::Level::debug, "delete", fmt::format("No mounts to stop for instance \"{}\"", name));
```

Comment:
Is this the only reason why the above would throw? I'm worried we're catching too broad here?

Reply:
I don't think there could be anything other than std::out_of_range thrown in this block.  However, there is probably some place some other exception can be thrown, so I'll tighten this up:)